docs/sources/shared/configuration.md

```diff
@@ -3927,6 +3927,9 @@ otlp_config:
 # CLI flag: -validation.enforced-labels
 [enforced_labels: <list of strings> | default = []]
 
+# Map of policies to stream selectors with a priority. Experimental.
```

Comment:
Is there a default for this setting?  (even if the default is nothing is set?)
Also, this could use an example in the description, since users might not know what a mapping should look like.

Reply:
No, default means no mapping and no policies assigned.
But good call, I'll improve the docs of this struct.

Reply:
Pushed a docs example, wdyt?

Reply:
Nice!  So glad you added the example, as that's pretty complicated.